kernelci/config/test.py

```diff
@@ -278,7 +278,7 @@ class RootFS(_YAMLObject):
     """Root file system model."""
 
     def __init__(self, fs_type, boot_protocol='tftp', root_type=None,
-                 prompt="/ #", params=None, ramdisk=None, nfs=None):
+                 prompt="/ #", params=None, ramdisk=None, nfs=None, disk=None):
```

Comment:
Shouldn't this be `diskfile` rather than `disk` to be more specific about how it's meant to be used?  I think `disk` could be also interpreted as a `/dev/sd*` block device or some other kind of physical storage medium.

Reply:
Thanks for the inputs. I have update disk with diskfile

Reply:
Thanks.  It's a shame the GitHub checks didn't pick this up, I'll see if we can improve on that too.